build_all/windows/build.cmd

```diff
@@ -68,10 +66,6 @@ if %build-platform% == x64 (
 )
 goto args-continue
 
-:arg-use-websockets
-set CMAKE_use_wsio=ON
-goto args-continue
-
 :arg-build-nuget
 shift
 if "%1" equ "" call :usage && exit /b 1
```

Comment:
These must be removed too:

build_all\windows\build.cmd:115:    cmake %build-root% -Duse_wsio:BOOL=%CMAKE_use_wsio%
build_all\windows\build.cmd:125:    cmake %build-root% -Duse_wsio:BOOL=%CMAKE_use_wsio% -G "Visual Studio 14 Win64"
build_all\windows\build.cmd:135:    cmake %build-root% -Duse_wsio:BOOL=%CMAKE_use_wsio% -G "Visual Studio 14 ARM"
build_all\windows\build.cmd:140:    cmake %build-root% -Drun_unittests:BOOL=ON -Duse_wsio:BOOL=%CMAKE_use_wsio%
build_all\windows\build.cmd:144:    cmake %build-root% -Drun_unittests:BOOL=ON -Duse_wsio:BOOL=%CMAKE_use_wsio% -G "Visual Studio 14 ARM"
build_all\windows\build.cmd:148:    cmake %build-root% -Drun_unittests:BOOL=ON -Duse_wsio:BOOL=%CMAKE_use_wsio% -G "Visual Studio 14 Win64"